prime-dotnet-webapi/Controllers/SitesController.cs

```diff
@@ -868,5 +868,30 @@ public async Task<ActionResult<IEnumerable<BusinessEvent>>> GetSiteBusinessEvent
 
             return Ok(ApiResponse.Result(events));
         }
+
+        // DELETE: api/Sites/{enrolleeId}/adjudication-documents/{documentId}
+        /// <summary>
+        /// Delete the site's adjudication document
+        /// </summary>
+        /// <param name="documentId"></param>
+        /// <returns></returns>
```

Comment:
The integration with Swagger will automatically show the request and response so we haven't been using the return annotations in the comments.